src/cloud/components/RateLimitAlert.tsx

```diff
@@ -29,14 +31,29 @@ import {CLOUD} from 'src/shared/constants'
 import {AppState} from 'src/types'
 import {LimitStatus} from 'src/cloud/actions/limits'
 import RateLimitAlertContent from 'src/cloud/components/RateLimitAlertContent'
-import WriteLimitAlertContent from 'src/cloud/components/WriteLimitAlertContent'
+
+import {notify} from 'src/shared/actions/notifications'
+import {writeLimitReached} from 'src/shared/copy/notifications'
+
+// Selectors
+import {shouldShowUpgradeButton} from 'src/me/selectors'
+import {dismissOverlay, showOverlay} from 'src/overlays/actions/overlays'
+import {UpgradeContent} from 'src/cloud/components/RateLimitAlertContent'
+
+import './RateLimitAlert.scss'
```

Comment:
not sure if this would need to be included in chronograf.scss?

Reply:
leave my modules alone! ;)